[
    {
        "type": "message",
        "user": "U1G62EYMN",
        "text": "Does anyone know of a good getting started tutorial or post? From installing to sending transactions and deploying a smart contract?",
        "ts": "1472285888.002441"
    },
    {
        "type": "message",
        "user": "U1P6JA98S",
        "text": "Hi, while doing make with latest hyperledger fabric source from github, it is exiting with result below",
        "ts": "1472298034.002442"
    },
    {
        "type": "message",
        "user": "U1P6JA98S",
        "text": "root@sonyshang:\/usr\/local\/gopkg\/src\/github.com\/hyperledger\/fabric# make\nBuilding docker base-image\nError: No such image, container or task: hyperledger\/fabric-baseimage:x86_64-0.0.11\nBUILD-CACHE: Pulling \"hyperledger\/fabric-baseimage:x86_64-0.0.11\" from dockerhub..\nPulling repository <http:\/\/docker.io\/hyperledger\/fabric-baseimage|docker.io\/hyperledger\/fabric-baseimage>\nTag x86_64-0.0.11 not found in repository <http:\/\/docker.io\/hyperledger\/fabric-baseimage|docker.io\/hyperledger\/fabric-baseimage>\nError: No such image, container or task: hyperledger\/fabric-baseimage:x86_64-0.0.11\nBUILD-CACHE: WARNING - Build-cache unavailable, attempting local build\nmake[1]: Entering directory '\/usr\/local\/gopkg\/src\/github.com\/hyperledger\/fabric\/images\/base'\nGenerating docker\nSending build context to Docker daemon 885.8 kB\nStep 1 : FROM ubuntu:trusty\ntrusty: Pulling from library\/ubuntu\n862a3e9af0ae: Pulling fs layer \n6498e51874bf: Pulling fs layer \n159ebdd1959b: Pulling fs layer \n0fdbedd3771a: Waiting \n7a1f7116d1e3: Waiting \nerror pulling image configuration: Get <https:\/\/dseasb33srnrn.cloudfront.net\/registry-v2\/docker\/registry\/v2\/blobs\/sha256\/4a\/4a725d3b3b1cc18c8cbd05358ffbbfedfe1eb947f58061e5858f08e2899731ee\/data?Expires=1472298733&amp;Signature=EJV75fbjSKmoK~pFhGK19po-3zQpjLXSWPMxJYKg-wti9e-6abeTbEhO89wj02mjq-rppAf83SdVbHmjBUCCE7vEPC~29gc5rO3BVhNkRwc0W3GRuaCmAS~eADB5T15bNS-eLyxUy~8XAnFGkNGpv48k9aXuXRF0QFeDyZurKfY_&amp;Key-Pair-Id=APKAJECH5M7VWIS5YZ6Q>: net\/http: TLS handshake timeout\nMakefile:33: recipe for target 'docker' failed\nmake[1]: *** [docker] Error 1\nmake[1]: Leaving directory '\/usr\/local\/gopkg\/src\/github.com\/hyperledger\/fabric\/images\/base'\nERROR: Build-cache could not be compiled locally\nMakefile:181: recipe for target 'build\/image\/base\/.dummy' failed\nmake: *** [build\/image\/base\/.dummy] Error 255",
        "ts": "1472298038.002443"
    },
    {
        "user": "U25JUR1SQ",
        "text": "<@U25JUR1SQ|avram> has joined the channel",
        "type": "message",
        "subtype": "channel_join",
        "ts": "1472303619.002444"
    },
    {
        "type": "message",
        "user": "U0KM61BCP",
        "text": "<@U1P6JA98S> are you building on AWS?",
        "ts": "1472305753.002445"
    },
    {
        "user": "U25K2EAQY",
        "text": "<@U25K2EAQY|alwaysanirudh> has joined the channel",
        "type": "message",
        "subtype": "channel_join",
        "ts": "1472307165.002446"
    },
    {
        "type": "message",
        "user": "U1P1ZV6RF",
        "text": "where can i report an issue with a failed deployment of fabric dev environment via vagrant as per readthedocs manual?",
        "ts": "1472309843.002447"
    },
    {
        "type": "message",
        "user": "U0KPFAZNF",
        "text": "<#C0YQ1NHGD|fabric-dev-env> is a good place within the slack",
        "ts": "1472309942.002448"
    },
    {
        "type": "message",
        "user": "U0KPFAZNF",
        "text": "note that there is a known issue introduced last night, the fix is in now its just being tested",
        "ts": "1472309966.002449"
    },
    {
        "user": "U25KDC62U",
        "text": "<@U25KDC62U|sh_balfas> has joined the channel",
        "type": "message",
        "subtype": "channel_join",
        "ts": "1472313571.002450"
    },
    {
        "user": "U25KGC892",
        "text": "<@U25KGC892|jmfranc> has joined the channel",
        "type": "message",
        "subtype": "channel_join",
        "ts": "1472316706.002451"
    },
    {
        "type": "message",
        "user": "U1D9R3WAX",
        "text": "hi im trying to run 4 peers with pbft using v0.5 of fabric and i get this error after deploying chaincode:\n^[[36m02:27:24.930 [consensus\/obcpbft] unicastOne -&gt; DEBU 5cf^[[0m could not send to replica 1: Message handler not found for receiver vp1\n^[[36m02:27:24.930 [consensus\/obcpbft] unicastOne -&gt; DEBU 5ce^[[0m could not send to replica 3: Message handler not found for receiver vp3\n^[[36m02:27:24.931 [consensus\/obcpbft] unicastOne -&gt; DEBU 5d0^[[0m could not send to replica 2: Message handler not found for receiver vp2\n0\n Replica 0 found incorrect signature in view-change message: Could not verify message from vp0",
        "ts": "1472351535.002452"
    }
]